app/abilities/base_ability.rb

```diff
@@ -0,0 +1,6 @@
+class BaseAbility
+  include CanCan::Ability
+
+  def initialize(user)
+  end
```

Comment:
I never when to add space between final `end` and previous `end`. And somehow I am glad that everyone seems to do it differently across the project :|